app/controllers/admin/call_for_papers_controller.rb

```diff
@@ -15,8 +15,8 @@ def create
       @call_for_paper = @conference.build_call_for_paper(call_for_paper_params)
 
       if @call_for_paper.save
-        redirect_to admin_conference_call_for_paper_path,
-                    notice: 'Call for papers successfully created.'
+        flash[:notice] = 'Call for papers successfully created.'
+        redirect_to admin_conference_call_for_paper_path
```

Comment:
Why splitting them in 2 lines?  
* Conformity: one style in all our code
* Clarity: each line does 1 specific thing, and it is clear what each line of code does
* Easier to read and understand
* Single style, as guideline, for new contributors

Part of our code has 1 single line with the redirect that includes the notice/alert message (app/controllers/admin/call_for_papers_controller.rb) and part of the code has them separated in 2 lines (https://github.com/openSUSE/osem/blob/master/app/controllers/admin/call_for_papers_controller.rb#L21). Some of the code is somewhere in between with just the redirect_to than inclues the message, but with a few break lines in between (https://github.com/openSUSE/osem/blob/master/app/controllers/admin/call_for_papers_controller.rb#L18). And that's even in one single file.

Part of the code has ***redirect_to admin_conference_call_for_paper_path, notice: 'text'*** and part of it has the same thing but with a parenthesis ***redirect_to(admin_conference_call_for_paper_path, notice: 'text')***

In my opinion, it would be nice to have it all coded the same way. 
And I think that separation of the message from the redirection code is easier to read and graps while browsing through lines of code.